tests/virt_autotest/virt_utils.pm

```diff
@@ -32,13 +32,20 @@ sub enable_debug_logging {
     turn_on_libvirt_debugging_log;
 
     # enable journal log with previous reboot
-    my $journald_conf_file = "/etc/systemd/journald.conf";
-    if (!script_run "ls $journald_conf_file") {
+    if (is_sle('>=15-SP6')) {
+        unless (get_var('AUTOYAST')) {
+            my $journald_conf_file = "/etc/systemd/journald.conf.d/01-virt-test.conf";
+            script_run("echo -e \"[Journal]\\\nStorage=persistent\" > $journald_conf_file");
+            script_run "systemd-analyze cat-config systemd/journald.conf | grep Storage";
```

Comment:
You use /etc/systemd/journald.conf.d/01-virt-test.conf as confi file, but systemd-analyze systemd/journald.conf. Is this expected ?

Reply:
Initially I did not notice it but verified on SLE, it worked well.  I just checked with the usuage and confirm this usage should be correct:
```
   systemd-analyze cat-config NAME|PATH...
       ... It will copy the contents of a config file and any drop-ins to standard output, using the usual systemd set of directories and rules for precedence.
       Each argument must be either an absolute path including the prefix (such as /etc/systemd/logind.conf or
       /usr/lib/systemd/logind.conf), or a name relative to the prefix (such as systemd/logind.conf).

       Example 15. Showing logind configuration

           $ systemd-analyze cat-config systemd/logind.conf
           # /etc/systemd/logind.conf
           ...
           [Login]
           NAutoVTs=8
           ...
           # /usr/lib/systemd/logind.conf.d/20-test.conf
           ... some override from another package
```